index.html

```diff
@@ -5,19 +5,19 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <meta name="description" content="QMK Configurator is an online tool used for easily creating firmware files for keyboards supported in qmk_firmware">
-    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <link rel="icon" href="/favicon.ico">
     <title>QMK Configurator</title>
     <style>
         .qmk-hidden-drag-n-drop {
           margin-left: -100vh;
         }
     </style>
  </head>
-  </head>
   <body>
     <noscript>
       <strong>We're sorry but configurator doesn't work properly without JavaScript enabled. Please enable it to continue.</strong>
     </noscript>
+    <script type="module" src="/src/main.js"></script>
```

Comment:
relying directly on /src? Isn't it bundled? i'm not familiar with vite.

Reply:
https://vitejs.dev/guide/#index-html-and-project-root

```
Vite treats index.html as source code and part of the module graph. It resolves <script type="module" src="..."> that references your JavaScript source code. Even inline <script type="module"> and CSS referenced via <link href> also enjoy Vite-specific features. In addition, URLs inside index.html are automatically rebased so there's no need for special %PUBLIC_URL% placeholders.
```

Reply:
This is required for ESM module stuff to work in browser for dev mode.

The bundling is done by rollup afaict. In the final dist it writes this:
```
   <script type="module" crossorigin src="/assets/index.dd94f9d5.js"></script>
   <link rel="modulepreload" href="/assets/vendor.61906764.js">
   <link rel="stylesheet" href="/assets/index.1270ddda.css">
```

Reply:
ok fair :)